dependencies.py

```diff
@@ -35,8 +35,9 @@
     subprocess.check_call(['git', 'submodule', 'init'])
     subprocess.check_call(['git', 'submodule', 'update'])
     logging.info("Dependencies installed")
-except subprocess.CalledProcessError:
-    logging.fatal("Git is not installed! Please install it!")
+except subprocess.CalledProcessError as e:
+    if e.returncode is 127:
```

Comment:
change to `if (e.returncode == 127):`. I got a syntax warning on Arch Linux because "is" is a literal.

Reply:
Good catch, even though I am suprised that that is possible in the first place

Reply:
It's fairly technical, but the gist of it is that any numeric literal is not guaranteed by the Python language specification to be identity-equal to a different instance of that number, i.e. `5 is 5` is not guaranteed. CPython has an optimization that numbers in the range [-5, 255] are singletons, so that `5 is 5` will return True in CPython. This is an exception however, as it is incompatible with other Python implementations- hence why comparing numbers with `==` rather than `is` is preferred.